index.html

```diff
@@ -2687,17 +2687,52 @@ <h1>Privacy Considerations</h1>
 	<section id="profiling-requesters">
 		<h2>Profiling of DID Resolution and Dereferencing Requesters</h2>
 		<p><a>DID resolvers</a> and <a>DID URL dereferencers</a> will be able to log requests
-			to their services for resolution and dereferencing. Over time, these logs could be used to track 
-			and profile the clients making requests for these services. To mitigate this privacy risk, 
-			clients should make such requests to services they trust, for example, 
-			because of an existing business relationship or because the service is running on 
-			infrastructure they control. Clients can also take steps to obfuscate their 
+			to their services for resolution and dereferencing. Over time, these logs could be used to track
+			and profile the clients making requests for these services. To mitigate this privacy risk,
+			clients should make such requests to services they trust, for example,
+			because of an existing business relationship or because the service is running on
+			infrastructure they control. Clients can also take steps to obfuscate their
 			requests to a service in order to limit the possibilities of correlation and profiling.
 		</p>
-	
+
 	</section>
 
-</section>	
+</section>
+
+<section class="appendix">
+	<h1>Relationship to Other Technologies</h1>
+
+	<p>
+One of the most common mechanisms to resolve an identifier to an addresses on
+the Internet is the global Domain Name System (DNS) described in [[?RFC1034]].
+The DNS and the processes and systems used to map a Domain Name to an Internet
+Protocol address is a common requirement for hosting a website.
+	</p>
+	<p>
+The [[[DID]]] specification introduced a new type of identifier that removed
+the dependency on the global Domain Name System and introduced the concept of
+an identifier resolution process that did not require the centralization of
+any part of the architecture. This new architecture allows the decentralized
+creation and management of a globally-resolveable identifier that combats
+identifier rent-seeking and censorship. It enables an individual to fully
+own and control their identifiers instead of renting the identifier from a
+third party.
+	</p>
+	<p>
+Individuals that are provided [=DID URLs=] use them in their software much like
+they use DNS-based URLs today. The software uses a [=DID resolver=] interface
+(defined in this specification) to determine the location of the resource being
+retrieved. The process of [=DID resolution=], much like the process of DNS
+resolution, is opaque to the individual and happens within software without any
+direct involvement needed of the individual.
```

Comment:
The diff is properly highlighted in the old interface, but not in the new. Two steps forward, three steps back, as is the GitHub way.
```suggestion
Individuals that acquire [=DID URLs=] use them in their software much like
they continue to use DNS-based URLs. The software uses a [=DID resolver=]
interface (defined in this specification) to determine the location of the
resources to be retrieved. The process of [=DID resolution=], much like the
process of DNS resolution, is opaque to the individual and happens within
the software without needing any direct involvement of the individual.
```